auth0/express-openid-connect
# Add cross domain iframe support for modern browsers

### Description

Fixing crossdomain iframe login/logout and attemptSilentAuth support. This can be enabled by using `session.cookie.sameSite = 'None'` and `session.cookie.secure = true` (you should run localhost on https for this see: https://auth0.com/docs/libraries/secure-local-development)

### References

fixes #315 

### Testing

To test login/logout from a cross domain iframe:
1. Run an https reverse proxy like caddy, eg `caddy reverse-proxy --from localhost:443 --to localhost:3000`
2. Run the iframe testing example with `baseUrl` set to `https://localhost`
3. Visit `http://localhost:3000/iframe` (running same app on a different domain to the iframe, which is on `https://localhost`)
4. Login and verify that the `auth_verification` transaction cookies have been removed
5. Logout and verify that the `appSession` cookie has been removed

### Checklist
